### PR TITLE
macro for exporting alerts used for notifications

### PR DESCRIPTION
**Pull request by dejii**
## What
Export alerts within specified date range.

### CODE REVIEW

**Comment by mateuszklimek** [comment]
@dejii is this one good and ready to be merged?

File: models/anomalies/re_data_alerts.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
+select
+    'anomaly' as type,
+    {{ re_data.clean_blacklist('table_name', ['"'], '') }} as model,
```
Comment (mateuszklimek) [inline models/anomalies/re_data_alerts.sql:3]: I think here BigQuery will have ` ticks instead of "

Comment (dejii) [reply models/anomalies/re_data_alerts.sql:3]: yeah, thanks for this. will add it to the list of characters.

File: models/anomalies/re_data_alerts.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    {{ ref('re_data_alerting') }}
+union all
+
+select
+    'schema_change' as type,
+    {{ re_data.clean_blacklist('table_name', ['"', '`'], '') }} as model,
+    {{ generate_schema_change_message('operation', 'column_name', 'prev_column_name', 'prev_data_type', 'data_type', 'detected_time') }} as message,
+    '' as value,
+    detected_time as time_window_end
+from {{ ref('re_data_schema_changes') }}
```
Comment (dejii) [inline models/anomalies/re_data_alerts.sql:17]: This model generates a formatted message for anomalies and schema changes. The notifications feature in the python project highlighted the need for centralizing this logic within dbt. Now same view on the observability UI can be seen directly by querying the DB 😄

https://github.com/re-data/re-data/issues/191

**Pull request merged**